packages/discovery-react-components/src/components/CIDocument/utils/__tests__/elementFromPoint.spec.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  elementFromPoint,
+  elementFromPointMs,
+  elementFromPointFallback,
+  MsDocument
+} from '../elementFromPoint';
+
+const clientX = 50;
+const clientY = 50;
+const className = 'field--rect';
+const stopElem = null;
+
+const classList = (list: string[]): any => {
+  return {
+    contains: (name: string): boolean => {
+      return list.includes(name);
+    }
+  };
+};
+
+const mockElements = [
+  { classList: classList(['mockName']), style: { pointerEvents: null } },
+  { classList: classList(['mockClass']), style: { pointerEvents: null } },
+  { classList: classList([className]), style: { pointerEvents: null } }
+];
+
+describe('elementFromPoint', () => {
+  it('elementsFromPoint', () => {
```

Comment:
Nit: Think of `describe` and `it` as parts of a sentence: 

```javascript
describe('CIDocument`, () => {
  it('renders properly with a contract', ...)
```

-> "CIDOcument renders properly with a contract".

You setup here doesn't work like that. Please fix.